Story-Chief/frontend-test-zenowatteeuw
# EventFilter: Changed filter to use .some. Now inclusive

## Description

Changed the event filtering to now be inclusive. Events will be included as soon as a single type is present.

## Closes issue(s)



## Testing Instructions

1. Open up the calendar
2. Choose an event with at least 2 types and note its types.
3. Set the event type one of the relevant types
4. Note that the event shows up despite having more types than the one selected

## Screenshots

![afbeelding](https://user-images.githubusercontent.com/24605774/215805461-91366018-c420-4296-94ba-a697c1dbf4f0.png)
